channels/uk.m3u

```diff
@@ -473,6 +549,8 @@ https://tennischannel-intl-samsung-uk.amagi.tv/playlist.m3u8
 https://timeline-samsung-uk.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="TJC.uk" tvg-name="TJC" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/cFlNhjV.png" group-title="",TJC (1080p)
 https://cdn-shop-lc-01.akamaized.net/Content/HLS_HLS/Live/channel(TJCOTT)/index.m3u8
+#EXTINF:-1 tvg-id="Tnt.uk" tvg-name="Tnt" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/CzuwTbL.png" group-title="",TNT
+http://stream.tvtap.live:8081/live/uk-tnt.stream/playlist.m3u8
```

Comment:
Originates from the US.

Reply:
If the link is for a uk specific channel it goes in uk.m3u not us.m3u as the channel would not be shown/streamed in the US.

Reply:
https://en.wikipedia.org/wiki/TNT_(American_TV_network)